pkg/buildqueue/buildqueue.go

```diff
@@ -122,7 +122,7 @@ func NewBuildQueue(name string) *BuildQueue {
 		name:             name,
 		buildStatus:      buildStatusMap{},
 		workerBuildQueue: make(chan Builder, buildQueueSize),
-		stopWorker:       make(chan struct{}, 0),
+		stopWorker:       make(chan struct{}),
```

Comment:
commit missing sign-off

Reply:
this code does not exist on master you will need to drop this commit and rebase against master